src/services/index.js

```diff
@@ -0,0 +1,13 @@
+import list from "../database/config/tokenlist";
+
+class listservices {
+
+    static AddToList(data) {
+        list.push(data);
+        return list;
+    }
+    static checklist(data) {
+        return list.includes(data);
+    }
+}
+export default listservices;
```

Comment:
export default class listservices and remove that one on line 13.
Remember to add new line